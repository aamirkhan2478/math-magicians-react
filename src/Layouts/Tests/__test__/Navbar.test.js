import renderer from 'react-test-renderer';
import Navbar from '../__mock__/Navbar';

it('renders navbar correctly', () => {
  const tree = renderer.create(<Navbar />).toJSON();
  expect(tree).toMatchSnapshot();
});
