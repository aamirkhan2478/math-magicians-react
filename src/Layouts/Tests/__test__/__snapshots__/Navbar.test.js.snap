// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header>
  <nav>
    <h4
      className="navbar-logo"
    >
      Math Magicians
    </h4>
    <ul
      className="nav"
    >
      <li
        className="nav-link"
      >
        <a
          aria-current="page"
          className="active"
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li
        className="nav-link"
      >
        <a
          className=""
          href="/calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
      </li>
      <li
        className="nav-link"
      >
        <a
          className=""
          href="/quote"
          onClick={[Function]}
        >
          Quote
        </a>
      </li>
    </ul>
  </nav>
</header>
`;

exports[`renders navbar correctly 1`] = `
<header>
  <nav>
    <h4
      className="navbar-logo"
    >
      Math Magicians
    </h4>
    <ul
      className="nav"
    >
      <li
        className="nav-link"
      >
        <a
          aria-current="page"
          className="active"
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li
        className="nav-link"
      >
        <a
          className=""
          href="/calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
      </li>
      <li
        className="nav-link"
      >
        <a
          className=""
          href="/quote"
          onClick={[Function]}
        >
          Quote
        </a>
      </li>
    </ul>
  </nav>
</header>
`;
